macros/adapters/default/sessions/snowplow_sessions_tmp.sql

```diff
@@ -66,12 +66,12 @@ prep AS (
         sum(time_engaged_in_s) as time_engaged_in_s,
 
         max(case when last_page_view_in_session = 1 then page_url else null end)
-            as exit_page_url,
+            as exit_page_url
         
         {%- for column in var('snowplow:pass_through_columns') %}
-        max(case when last_page_view_in_session = 1 then {{column}} else null end)
+        , max(case when last_page_view_in_session = 1 then {{column}} else null end)
```

Comment:
i didn't notice this in the original PR, but should this be `last_page_view_in_session` or `first_page_view_in_session`? I imagine first, right?

Reply:
We include both `first_` and `last_` for the custom column, which is . The `prep` CTE here gives us access to `last_`, whereas we can just pull `first_` from the first page view (`a` alias) in the join below.